fly/integration/sync_test.go

```diff
@@ -61,17 +61,9 @@ var _ = Describe("Syncing", func() {
 
 			flyVersion = fmt.Sprintf("%d.%d.%d", major, minor, patch+1)
 		})
-		It("downloads and replaces the currently running executable", func() {
-			flyCmd := exec.Command(flyPath, "-t", targetName, "sync")
-
-			sess, err := gexec.Start(flyCmd, GinkgoWriter, GinkgoWriter)
-			Expect(err).NotTo(HaveOccurred())
-
-			<-sess.Exited
-			Expect(sess.ExitCode()).To(Equal(0))
 
-			expected := []byte("this will totally execute")
-			expectBinaryToMatch(flyPath, expected[:8])
+		It("downloads and replaces the currently running executable", func() {
+			downloadAndReplaceExecutable(flyPath,"sync", "-c", atcServer.URL())
```

Comment:
minor nit: `go fmt` - if possible I'd recommend enabling auto-`fmt`-on-save in your editor